.github/workflows/cross-version-tests.yml

```diff
@@ -75,8 +75,8 @@ jobs:
           hash=$(echo -n "$INSTALL_COMMAND" | sha256sum)
           echo "::set-output name=key::$ImageOS-$ImageVersion-wheels-$date-$hash"
       - uses: actions/cache@v2
-        # We only cache wheels that take long (> 20 min) to build
-        if: ${{ contains('pyspark, catboost', matrix.package) && matrix.version == 'dev' }}
+        # We only cache wheels that take long (> 10 min) to build
+        if: ${{ contains('pyspark, catboost, scikit-learn', matrix.package) && matrix.version == 'dev' }}
```

Comment:
Added `sklearn` to cache-required packages sicne it takes long to build sklearn from the source.